fiaas_deploy_daemon/deployer/kubernetes/role_binding.py

```diff
@@ -0,0 +1,100 @@
+import logging
+
+from k8s.models.common import ObjectMeta
+from k8s.models.role_binding import RoleBinding, RoleRef, Subject
+from k8s.client import NotFound
+from fiaas_deploy_daemon.specs.models import AppSpec
+from fiaas_deploy_daemon.deployer.kubernetes.owner_references import OwnerReferences
+from fiaas_deploy_daemon.tools import merge_dicts
+
+LOG = logging.getLogger(__name__)
+
+
+class RoleBindingDeployer:
+    def __init__(self, config, owner_references):
+        self._owner_references: OwnerReferences = owner_references
+        self._list_of_roles = config.list_of_roles
+        self._list_of_cluster_roles = config.list_of_cluster_roles
+
+    def deploy(self, app_spec: AppSpec, labels):
+        custom_annotations = {}
+        custom_labels = labels
+        custom_labels = merge_dicts(app_spec.labels.role_binding, custom_labels)
+        custom_annotations = merge_dicts(app_spec.annotations.role_binding, custom_annotations)
+        role_bindings = RoleBinding.find(name=app_spec.name, namespace=app_spec.namespace)
```

Comment:
Might want to comment here that `name` refers to the `app` label value. It didn't make sense to me until I looked up the implementation of the find function.

Reply:
Added a comment 😄